content/v2.0/reference/cli/influx/bucket/create.md

```diff
@@ -25,6 +25,6 @@ influx bucket create [flags]
 | `-n` | `--name`         | Bucket name                                                    | string      | `INFLUX_BUCKET_NAME`  |
 | `-o` | `--org`          | Organization name                                              | string      | `INFLUX_ORG`          |
 |      | `--org-id`       | Organization ID                                                | string      | `INFLUX_ORG_ID`       |
-| `-r` | `--retention`    | Duration bucket will retain data (0 is infinite, default is 0) | duration    |                       |
+| `-r` | `--retention`    | Duration the bucket will retain data (0 is infinite, default is 0). Valid units are nanoseconds (`ns`), microseconds (`us` or `µs`), milliseconds (`ms`), seconds (`s`), minutes (`m`), hours (`h`), days (`d`), and weeks (`w`). | duration    |                       |
```

Comment:
I think we may need to do something differently here. This is a lot of content inside of a table cell and is likely going to do some less than ideal things to the table formatting. I think we should pull the valid units statement out of the table and include them in a separate note.

Reply:
Thanks, @sanderson - updated bucket create, bucket update, and setup.